19.md

```diff
@@ -65,5 +63,6 @@ These possible standardized `TLV` types are indicated here:
 
 ## Notes
 
+- Clients should still accept keys in both hex and npub format for now, and convert internally.
```

Comment:
```suggestion
```

I prefer to not have this otherwise people will think using npubs in `p` tags is ok, for example.

Reply:
I agree, but just moved the original sentence.